epfl-restauration.php

```diff
@@ -2,72 +2,178 @@
 /**
  * Plugin Name: EPFL Restauration
  * Description: provides a shortcode to display cafeterias and restaurant menus offers
- * Version: 0.3
- * Author: Lucien Chaboudez
- * Contributors:
- * License: Copyright (c) 2019 Ecole Polytechnique Federale de Lausanne, Switzerland
+ * Version: 1.0
+ * Author: Jérémy Wolff
+ * Contributors: Lucien Chaboudez, Quentin Estoppey
+ * License: Copyright (c) 2021 Ecole Polytechnique Federale de Lausanne, Switzerland
  **/
 
+define("REMOTE_SERVER_TIMEOUT", 10);  // time to wait until we consider the remote server out of the game
+define("REMOTE_SERVER_SSL", true);  // force the server to be https certified
+define("LOCAL_CACHE_NAME", 'EPFL_RESTAURATION');  // the option and transient name for the caching
+define("LOCAL_CACHE_TIMEOUT", 10 * 60);  // cache time validity, in seconds
+
+// Configuration with sections
+$ini_array_sections = parse_ini_file("menus.ini", true);
 
 function epfl_restauration_process_shortcode( $atts, $content = null ) {
 
     global $wp;
+    global $ini_array_sections;
 
     $atts = shortcode_atts( array(
         'params' => '',
         'type' => 'menu', // can be 'menu' or 'schedule'
     ), $atts );
 
-    /* We remove ? at the beginning if any*/
+    /* We remove ? at the beginning if any */
     $params = preg_replace('/^\?/', '', $atts['params']);
     $type = sanitize_text_field($atts['type']);
 
+    /* Including CSS file*/
+    wp_enqueue_style( 'epfl_restauration_style', plugin_dir_url(__FILE__).'css/style.css', [], '2.1');
+
+    $images_path = get_option("home") . "/wp-content/plugins/epfl-restauration/images/";
```

Comment:
je me sentirais plus serein avec
```suggestion
    $images_path = plugin_dir_url(__FILE__) . "images/";
```